Masterlist Updates

MacSplody opened the pull request.

MacSplody committed:
Masterlist Addition - Beyond Skyrim Bruma

Beyond Skyrim Bruma
- Dirty plugin info, crc.
- bash tags
BS DLC integration patch
- incompatible with DLC integration mod.
- added bash tags.

MacSplody committed:
Addendum

-Add crc for cleaned Bs Bruma plugin.

MacSplody committed:
Addendum #2

USSEP
- Added Cleaned entry

BS Bruma
- Added second CRC, when cleaning you can end up with 3 disabled null entries being referenced. Not sure what is causing this, but as a guess it could be because it's not a base skyrim entry being ref in the first place.

Ortham commented on masterlist.yaml:
```diff
@@ -382,8 +382,50 @@ plugins:
+        util: SSEEdit v3.2.1
+  - name: 'BSHeartland.esm'
+    dirty:
+    - <<: *dirtyPlugin
```
This indentation might work, but it's inconsistent, could you indent by another two spaces?

MacSplody replied:
Okay I think its the editor I'm using it's not great for yaml.

Ortham commented on masterlist.yaml:
```diff
@@ -382,8 +382,50 @@ plugins:
+      - crc: 0x6F103898
+        util: SSEEdit v3.2.1
+    tag:
+    - Actors.ACBS 
```
Same as above, please be consistent about indentation.

MacSplody committed:
Indentation Corrections

MacSplody committed:
Update Entries

DLCIntegration
- Added missing bash suggested tags.
- Added Clean plugin info
BS Bruma
- Rearranged Entry headings for consistancy.
